content/en/docs/concepts/cluster-administration/system-metrics.md

```diff
@@ -98,6 +98,14 @@ Take metric `A` as an example, here assumed that `A` is deprecated in 1.n. Accor
 
 If you're upgrading from release `1.12` to `1.13`, but still depend on a metric `A` deprecated in `1.12`, you should set hidden metrics via command line: `--show-hidden-metrics=1.12` and remember to remove this metric dependency before upgrading to `1.14`
 
+## Disable Accelerator Metrics
+
+Since Kubernetes release 1.9, the kubelet has been collecting Accelerator metrics through cadvisor. To collect such metrics, for accelerators such as NVIDIA GPUs, Kubelet held an open handle on that driver. This meant that to perform infrastructure changes (e.g: Update the driver), a cluster administrator needed to stop the kubelet agent.
+
+Going forward, the responsibility for collecting Accelerator metrics has shifted towards the vendor rather than kubelet. The vendor is expected to provide a container collecting the metrics and exposing them to the metrics service (e.g: Prometheus).
```

Comment:
```suggestion
As of v1.19, the responsibility for collecting accelerator metrics belongs to the vendor rather than the kubelet. Vendors must provide a container that collects the metrics and exposes them to the metrics service (for example, Prometheus).
```

Reply:
I'm going to reword to not provide a reference to v1.19, the responsibility didn't shift in v1.19 :D

Reply:
Fixed, thanks!